concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -477,21 +486,116 @@ jobs:
           params:
             file: generated-cf-secrets/cf-secrets.yml
 
+
+  - name: pre-deploy
+    plan:
+      - aggregate:
+          - get: pipeline-trigger
+            passed: ['generate-secrets']
+            trigger: true
+          - get: paas-cf
+            passed: ['generate-secrets']
+          - get: cf-secrets
+            passed: ['generate-secrets']
+          - get: bosh-CA
+            passed: ['generate-secrets']
+      - try:
+          task: wait-for-availability-test
+          config:
+            platform: linux
+            inputs:
+              - name: bosh-CA
+              - name: paas-cf
+              - name: cf-secrets
+              - name: pipeline-trigger
+            params:
+              SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
+              CF_ADMIN: admin
+            image: docker:///governmentpaas/cf-cli
+            run:
+              path: sh
+              args:
+                - -e
+                - -u
+                - -c
+                - |
+                  ./paas-cf/concourse/scripts/import_bosh_ca.sh
+
+                  CF_PASS=$(awk '/uaa_admin_password/ {print $2}' cf-secrets/cf-secrets.yml)
```

Comment:
Like https://github.com/alphagov/paas-cf/pull/321#discussion_r68070427 can we use `val_from_yaml.rb` here where we're using a container that has Ruby available?

(sorry, missed this yesterday)